### DISCUSSION TITLE
Subgender confusion

### DISCUSSION THREAD

**Discussion by FeralBloom** [ErinaSugino/Starbound-SBR-Mod#27] [Trouble Shooting] [answered]
I've messed around with the config a bunch but I can't seem to figure out how to make it so that you can change to futanari or cuntboy regardless of what your base gender is, could someone help me out?

**Comment by ErinaSugino** [comment, marked as answer]
What you are looking for is in the `sxb_plugin.subgender.config`. This config is really simple, as it only has two settings:
`genderLimitations`, which is a list that actually determines which sub-genders require which main genders. Clearing that will remove the restrictions.
`statuseffects`, which is a table of lists determining which statuseffects temporarily apply which sub-gender. By default, the `equipped_futamode` statuseffect from Lustbound's Futapatch back item will try to apply both futanari and cuntboy, because usually only one of the two succeeds due to the gender limitations. If you remove those, however, this item will now apply both sub-genders with the latter (cuntboy) always overriding the former (futanari), so you might want to change that too.

As always when editing config files, make sure you use proper, valid JSON syntax as otherwise stuff will break.
